Gemfile.jruby-2.5.lock.release

```diff
@@ -461,12 +464,13 @@ GEM
       logstash-filter-json
       logstash-input-generator
       logstash-output-file
-    logstash-output-elastic_app_search (1.0.0)
+    logstash-output-elastic_app_search (1.1.0)
       logstash-codec-plain
       logstash-core-plugin-api (~> 2.0)
-    logstash-output-elasticsearch (10.5.1-java)
+    logstash-output-elasticsearch (10.6.0-java)
```

Comment:
Per discussion with @jsvd - until we push the fix which currently breaks the build with the lastest version of the ES output, let's keep the previous version for now to have a green build on 7.9. 
```suggestion
    logstash-output-elasticsearch (10.5.1-java)
```